apps/frontend/src/modules/auth/graphql/queries/viewer.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from "@apollo/client";
+
+export const VIEWER = gql`
+    query viewer {
+        viewer {
+            id
+            email
+            firstName
+            lastName
+            isStaff
+            isActive
+            organization {
+                ...AuthOrganizationFragment
```

Comment:
@AfeezGL can you move this `viewer` query to `users` modules. `viewer` should not be part of `auth` because it has access to more user data.